### PR TITLE
Added python jenkins jobs

### PR DESCRIPTION
**Pull request by BasileoLuv**
Instruction will be later

### CODE REVIEW

File: jnk/jobs/simple-python-service-master-release/config.xml [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+  <blockBuildWhenUpstreamBuilding>false</blockBuildWhenUpstreamBuilding>
+  <jdk>JDK 8</jdk>
+  <triggers/>
+  <concurrentBuild>false</concurrentBuild>
+  <builders>
+    <hudson.tasks.Shell>
+      <command>git checkout master
+git branch
+currrelease=$(grep version setup.py | cut -d= -f2 | cut -d- -f1 | sed &apos;s/^.//&apos;)
+snapshot_version=$versionandtimestamp
```
Comment (savermyas) [inline jnk/jobs/simple-python-service-master-release/config.xml]: where the variable $versionandtimestamp is coming from?

File: jnk/jobs/simple-python-service-master-release/config.xml [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+wget http://nexus:8081/repository/pypi/packages/python-calculator/${release_version}-SNAPSHOT/Python-Calculator-${snapshot_version}.tar.gz
+tar xvzf Python-Calculator-${snapshot_version}.tar.gz
+rm Python-Calculator-${snapshot_version}.tar.gz
+</command>
+    </hudson.tasks.Shell>
+    <hudson.tasks.Shell>
+      <command>cd Python-*
+currsnapshot=$(grep version setup.py | cut -d= -f2 | cut -d- -f1 | sed &apos;s/^.//&apos;)
+echo $currsnapshot
+echo $currrelease
```
Comment (savermyas) [inline jnk/jobs/simple-python-service-master-release/config.xml:66]: $currelease will be empty on current step

File: jnk/jobs/update version/config.xml [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  <disabled>false</disabled>
+  <blockBuildWhenDownstreamBuilding>false</blockBuildWhenDownstreamBuilding>
+  <blockBuildWhenUpstreamBuilding>false</blockBuildWhenUpstreamBuilding>
+  <jdk>(System)</jdk>
+  <triggers/>
+  <concurrentBuild>false</concurrentBuild>
+  <builders>
+    <hudson.tasks.Shell>
+      <command>git checkout dev
+oh=$(grep version setup.py | cut -d= -f2 | cut -d- -f1 | sed &apos;s/^.//&apos;)
```
Comment (savermyas) [inline jnk/jobs/update version/config.xml]: what is "oh" name meaning?

File: jnk/jobs/update version/config.xml [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+major_version=$(echo $oh| cut -d. -f1)
+minor_version=$(echo $oh | cut -d. -f2)
+hotfix_version=$(echo $oh | cut -d. -f3)
+echo &quot;${major_version}.${minor_version}.$(( ${hotfix_version}+1))&quot;
+funk=&quot;${major_version}.${minor_version}.$(( ${hotfix_version}+1))&quot;
+sed &quot;s/version=.*/version=&apos;$funk-SNAPSHOT&apos;,/g&quot;  setup.py &gt;2.txt
```
Comment (savermyas) [inline jnk/jobs/update version/config.xml]: `2.txt` is not a good name for temporary file

**Review by savermyas — CHANGES_REQUESTED** [review]
please, update PR

File: jnk/jobs/simple-python-service-master-release/config.xml (hunk 1 above)
Comment (BasileoLuv) [reply jnk/jobs/simple-python-service-master-release/config.xml]: From jenkins paramètres

File: jnk/jobs/simple-python-service-master-release/config.xml (hunk 2 above)
Comment (BasileoLuv) [reply jnk/jobs/simple-python-service-master-release/config.xml:66]: Fixed

File: jnk/jobs/update version/config.xml (hunk 3 above)
Comment (BasileoLuv) [reply jnk/jobs/update version/config.xml]: Replaced

File: jnk/jobs/update version/config.xml (hunk 4 above)
Comment (BasileoLuv) [reply jnk/jobs/update version/config.xml]: Replaced

File: jnk/jobs/simple-python-service-master-release/config.xml (hunk 1 above)
Comment (savermyas) [reply jnk/jobs/simple-python-service-master-release/config.xml]: why wouldn't you just give it the name "snapshot_version"?

Comment (BasileoLuv) [reply jnk/jobs/simple-python-service-master-release/config.xml]: Cause it also timestamp

File: jnk/jobs/simple-python-service-master-release/config.xml [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+  <blockBuildWhenDownstreamBuilding>false</blockBuildWhenDownstreamBuilding>
+  <blockBuildWhenUpstreamBuilding>false</blockBuildWhenUpstreamBuilding>
+  <jdk>JDK 8</jdk>
+  <triggers/>
+  <concurrentBuild>false</concurrentBuild>
+  <builders>
+    <hudson.tasks.Shell>
+      <command>git checkout master
+currrelease=$(grep version setup.py | cut -d= -f2 | cut -d- -f1 | sed &apos;s/^.//&apos;)
+currsnapshot=$(grep version setup.py | cut -d= -f2 | cut -d- -f1 | sed &apos;s/^.//&apos;)
```
Comment (savermyas) [inline jnk/jobs/simple-python-service-master-release/config.xml]: why do you need it here?

File: jnk/jobs/simple-python-service-master-release/config.xml [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+echo $currsnapshot
+echo $currrelease
+
+if [  $currsnapshot &lt;  $currrelease ]
+then 
+  exit 1
+else 
+  git merge $(cat newcommit.txt)
+  git tag $(cat newcommit.txt)
```
Comment (savermyas) [inline jnk/jobs/simple-python-service-master-release/config.xml]: tag must contain release version, not commit SHA

File: jnk/jobs/update version/config.xml [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+  <builders>
+    <hudson.tasks.Shell>
+      <command>git checkout dev
+var=$(grep version setup.py | cut -d= -f2 | cut -d- -f1 | sed &apos;s/^.//&apos;)
+major_version=$(echo $var| cut -d. -f1)
+minor_version=$(echo $var | cut -d. -f2)
+hotfix_version=$(echo $var | cut -d. -f3)
+echo &quot;${major_version}.${minor_version}.$(( ${hotfix_version}+1))&quot;
+funk=&quot;${major_version}.${minor_version}.$(( ${hotfix_version}+1))&quot;
+sed &quot;s/version=.*/version=&apos;$funk-SNAPSHOT&apos;,/g&quot;  setup.py &gt;good_name_for_temporary_file.txt
```
Comment (savermyas) [inline jnk/jobs/update version/config.xml]: try to use `mktemp`

**Review by savermyas — CHANGES_REQUESTED** [review]
a few comments left

File: jnk/jobs/simple-python-service-master-release/config.xml (hunk 5 above)
Comment (BasileoLuv) [reply jnk/jobs/simple-python-service-master-release/config.xml]: ```sed "s/version=.*/version='$currsnapshot',/g"  setup.py >2.txt```

Comment (BasileoLuv) [reply jnk/jobs/simple-python-service-master-release/config.xml]: ```sed "s/version=.*/version='$currsnapshot',/g" setup.py > good_name_for_temporary_file.txt```

File: jnk/jobs/simple-python-service-master-release/config.xml (hunk 6 above)
Comment (BasileoLuv) [reply jnk/jobs/simple-python-service-master-release/config.xml]: ```git tag $(echo $currsnapshot)```
Its good choice?

File: jnk/jobs/simple-python-service-master-release/config.xml (hunk 2 above)
Comment (savermyas) [reply jnk/jobs/simple-python-service-master-release/config.xml:66]: How?

File: jnk/jobs/simple-python-service-master-release/config.xml (hunk 5 above)
Comment (savermyas) [reply jnk/jobs/simple-python-service-master-release/config.xml]: you reassign its value in line number 67, so this value is not used anywhere. why do you need it?

File: jnk/jobs/simple-python-service-master-release/config.xml (hunk 1 above)
Comment (savermyas) [reply jnk/jobs/simple-python-service-master-release/config.xml]: OK, but you assign it to the variable `snapshot_version` anyway

File: jnk/jobs/simple-python-service-master-release/config.xml [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+currsnapshot=$(grep version setup.py | cut -d= -f2 | cut -d- -f1 | sed &apos;s/^.//&apos;)
+echo $currsnapshot
+echo $currrelease
+
+if [  $currsnapshot &lt;  $currrelease ]
+then 
+  exit 1
+else 
+  git merge $(cat newcommit.txt)
+  git tag $(echo $currsnapshot)
```
Comment (savermyas) [inline jnk/jobs/simple-python-service-master-release/config.xml]: Not snapshot, but release

**Review by savermyas — CHANGES_REQUESTED** [review]
Issues hasn't been fixed still

**Review by savermyas — APPROVED** [review]

**Pull request merged**